librz/bin/format/java/class_bin.c

```diff
@@ -1046,6 +1050,37 @@ static char *add_class_name_to_name(char *name, char *classname) {
 	return strdup(name);
 }
 
+static char *demangle_java_and_free(char *mangled) {
+	if (!mangled) {
+		return NULL;
+	}
+	if (!strncmp(mangled, "unknown_", strlen("unknown_"))) {
```

Comment:
rz_str_startswith()

Reply:
`rz_str_startswith` is not optimized by the compiler, that one is.